src/context/data-context.js

```diff
@@ -0,0 +1,76 @@
+import { createContext, useContext, useReducer } from "react";
+import { getDatabase, ref, get } from "firebase/database";
+import { initializeApp } from "firebase/app";
+import { useEffect } from "react";
+
+const DataContext = createContext();
+
+const DataProvider = ({ children }) => {
+   var firebaseConfig = {
+      // The value of `databaseURL` depends on the location of the database
+      databaseURL:
+         "https://hhquiz-4ef5c-default-rtdb.asia-southeast1.firebasedatabase.app",
+   };
+
+   const app = initializeApp(firebaseConfig);
+
+   //fetching quiz data and categories from firebase realtime database
```

Comment:
remove commented code here as well

Reply:
Kept this comment for better understanding of the code.